src/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+set(SRC
+	main.cpp
+	notificationservice.cpp
+	weatherservice.cpp
+	mediaservice.cpp
+	batteryservice.cpp
+	screenshotservice.cpp
+	timeservice.cpp
+	service.cpp
+	characteristic.cpp
+	descriptor.cpp
+	application.cpp
+	advertisement.cpp
+	bluezmanager.cpp
+	ancs.cpp
+	ancs_notification.cpp)
+set(HEADERS
+	notificationservice.h
+	weatherservice.h
+	mediaservice.h
+	batteryservice.h
+	screenshotservice.h
+	timeservice.h
+	service.h
+	characteristic.h
+	descriptor.h
+	common.h
+	application.h
+	advertisement.h
+	bluezmanager.h
+	ancs_protocol_constants.h
+	ancs.h
+	ancs_notification.h)
+
+add_library(asteroid-btsyncd ${SRC} ${HEADERS})
```

Comment:
While many asteroid apps require them to be a library due to `mapplauncherd`. The btsyncd daemon doesn't use `mapplauncherd` so it can be a normal executable (`add_executable`). Also, please don't forget to the two lines below.